index.html

```diff
@@ -157,10 +157,28 @@ <h5 class="about-heading">About Me</h5>
                     </ul>
                 </li>
             </ul>
-            
         </section>
-             
     </section>
-
+    <footer >
```

Comment:
- [ ] Kindly make sure you use semantic tags on your page. the better your HTML is structured the better SEO will be.
- [ ] The `footer` tag doesn't have any semantic meaning being here.
`You may consider removing that.` also you can check [here](https://www.w3schools.com/tags/tag_footer.asp) when to use it